client/src/QnA/QnAList.jsx

```diff
@@ -22,13 +27,15 @@ function QnAList({ quests, product, grabQuestions }) {
             grabQuestions={grabQuestions}
           />
         ))}
-      {questButton ? null : (
-        <form onSubmit={submitHandler}>
-          <input type="submit" value="Show More Answered Questions" />
-        </form>
-      )}
+      <div className="question-buttons" data-testid="question-buttons">
+        {questButton ? null : (
+          <form onSubmit={submitHandler}>
+            <input type="submit" value="Show More Answered Questions" />
+          </form>
+        )}
+        {questButton ? null : (<button type="button" onClick={allQuestionHandler}>Show All Questions</button>)}
+      </div>
```

Comment:
In QnAList component, on line 32 you have a conditional rendering of a "show more answered questions". Is there a particular reason for using a form and input tag over just a button? There's also a button right below it with the same conditional to show all questions; I'm a little confused what roles each is playing in the widget, but that could also be because I'm only seeing this snippet (is there a way to see the entire file on github, not just the changes?)